lib/logstash/outputs/elasticsearch/common.rb

```diff
@@ -280,8 +280,11 @@ def safe_bulk(actions)
       end
     end
 
-    def supports_dlq?
-      respond_to?(:execution_context) && execution_context.respond_to?(:dlq_writer)
+    def dlq_enabled?
+      # TODO there should be a better way to query if DLQ is enabled
+      # See more in: https://github.com/elastic/logstash/issues/8064
+      respond_to?(:execution_context) && execution_context.respond_to?(:dlq_writer) &&
+        !execution_context.dlq_writer.inner_writer.is_a?(::LogStash::Util::PluginDeadLetterQueueWriter)
```

Comment:
Shouldn't this be `DummyDeadLetterQueueWriter`?

Reply:
🤦‍♂️

Reply:
changed, good catch